docs/reference/modules/cluster/remote-clusters-security.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[remote-clusters-security]]
+=== Configure remote clusters with security
+To use {ccr} or {ccs} with secured clusters, all clusters must have the {es} 
+security features enabled.
+
+The local cluster must be allowed to connect to the remote clusters, which means 
+that the certificate authority (CA) used to sign the keystore of the local
+cluster must be trusted by the remote clusters.
```

Comment:
That's a better way of explaining! How about this revision?
>All connected clusters must trust one another and be mutually authenticated at the TLS level. This means that the local cluster trusts the certificate authority (CA) of the remote cluster, and the remote cluster trusts the CA of
the local cluster.

Reply:
Yes that reads better. Please also see my [other comment](https://github.com/elastic/elasticsearch/pull/77043#discussion_r707839179) for more details about mutual trust.

Reply:
Thanks for that explanation @ywangd -- it's very helpful. Based on your other comment, I'm expanding this section a bit: 
>All connected clusters must trust one another and be mutually authenticated at the TLS level. This means that the local cluster trusts the certificate authority (CA) of the remote cluster, and the remote cluster trusts the CA of
the local cluster. When establishing a connection, all nodes will verify certificates from the nodes they are connecting to. This mutual trust is required to connect a remote cluster, because all connected nodes effectively form a single security domain.